lib/datadog/tracing/flush.rb

```diff
@@ -18,7 +18,7 @@ def consume!(trace_op)
         end
 
         def full_flush?(trace_op)
-          trace_op && trace_op.sampled? && trace_op.finished?
```

Comment:
Hmmm, why `trace_op.sampled?` omitted? Trying to figure out the purpose of this.